canvas.go

```diff
@@ -59,12 +87,20 @@ func (s *service) usage(canvas *svg.SVG) {
 		s.point.Y,
 		"#serviceBlock",
 		fmt.Sprintf(`id="%s"`, s.name))
-	canvas.Image(
-		s.point.X+serviceBlockSize/2-iconSize/2,
-		s.point.Y+serviceBlockSize/2-iconSize/2,
-		iconSize,
-		iconSize,
-		s.iconUrl)
+	if s.iconSrc != "" {
+		canvas.Use(
+			s.point.X+serviceBlockSize/2-iconSize/2,
+			s.point.Y+serviceBlockSize/2-iconSize/2,
+			fmt.Sprintf("#icon-%s", sanitizeSelector(s.charmPath)),
```

Comment:
This was poorly named (and has been removed anyway).  These aren't actually selectors, but URIs pointing to defined elements elsewhere in the document (similar to using `<a href="#top">top</a>`).